File: analyzers/tests/SonarAnalyzer.UnitTest/TestCases/TabCharacter.cs
using System.Collections.Generic;

namespace Tests.Diagnostics
{
    public class TabCharacter
    {
        public TabCharacter()
        {
            var tabs = "	"; // Noncompliant
            var tabs2 = "		";
            // some more tabs: "		"
        }
    }
}

File: analyzers/tests/SonarAnalyzer.Rules.Tests/Cases/TabCharacter.vb
Imports System.Collections.Generic

Namespace Tests.Diagnostics
    Public Class TabCharacter

        Public Sub New()
            Dim tabs = "	" ' Noncompliant {{Replace all tab characters in this file by sequences of white-spaces.}}
            Dim tabs2 = "		"
            ' some more tabs: "		"
        End Sub
    End Class
End Namespace

